axis.go

```diff
@@ -415,6 +415,76 @@ func (DefaultTicks) Ticks(min, max float64) (ticks []Tick) {
 	return
 }
 
+// AltTicks is suitable for the Tick.Marker field of an Axis,
+// it returns an alternative to the default set of tick marks.
+type AltTicks struct{}
+
+var _ Ticker = AltTicks{}
+
+// Ticks returns Ticks in a specified range
+func (AltTicks) Ticks(min, max float64) (ticks []Tick) {
+	const suggestedTicks = 3
+
+	var labels []float64
+
+	labels = nil
```

Comment:
that isn't needed. `var labels []float64` already initializes `labels` to `nil`.